scripts/init.sh

```diff
@@ -20,24 +20,46 @@
 
 # Command-line options that can be set in /etc/default/influxdb.  These will override
 # any config file values. Example: "-join http://1.2.3.4:8086"
+DEFAULT=/etc/default/influxdb
+
+# Daemon options
 INFLUXD_OPTS=
 
+# Process name ( For display )
+NAME=influxdb
+
+# User and group
 USER=influxdb
 GROUP=influxdb
 
-if [ -r /lib/lsb/init-functions ]; then
-    source /lib/lsb/init-functions
+# Daemon name, where is the actual executable
+# If the daemon is not there, then exit.
+DAEMON=/opt/influxdb/influxd
+[ -x $DAEMON ] || exit 5
+
+# Configuration file
+CONFIG=/etc/opt/influxdb/influxdb.conf
+
+# PID file for the daemon
+PIDFILE=/var/run/influxdb/influxd.pid
+PIDDIR=`dirname $PIDFILE`
+if [ ! -d "$PIDDIR" ]; then
+    mkdir -p $PIDDIR
+    chown $GROUP:$USER $PIDDIR
 fi
 
-DEFAULT=/etc/default/influxdb
+# Max open files
+OPEN_FILE_LIMIT=65536
 
-if [ -r $DEFAULT ]; then
-    source $DEFAULT
+if [ -r /lib/lsb/init-functions ]; then
+    source /lib/lsb/init-functions
```

Comment:
This is still working for RedHat flavours as you can see here [init script test](http://pastebin.com/wQnkc4e0)

Btw, you can check in the [current raw init file](https://raw.githubusercontent.com/influxdb/influxdb/master/scripts/init.sh) that we already load /lib/lsb/init-functions if exists, is something that this PR does not change.